test/dlc_tests/benchmark/ec2/pytorch/inference/test_performance_pytorch_inference.py

```diff
@@ -36,7 +38,7 @@ def ec2_performance_pytorch_inference(image_uri, processor, ec2_connection, regi
     commit_info = os.getenv("CODEBUILD_RESOLVED_SOURCE_VERSION")
     # Run performance inference command, display benchmark results to console
     container_name = f"{repo_name}-performance-{image_tag}-ec2"
-    log_file = f"inference_benchmark_results_{commit_info}_{time_str}.log"
+    log_file = f"synthetic_{commit_info}_{time_str}.log"
```

Comment:
Doesn't using the same log-file name for multiple images cause them to overwrite one another, when the tests are run in parallel?